qcengine/tests/test_harness_canonical.py

```diff
@@ -64,6 +64,35 @@ def test_compute_gradient(program, model):
     assert "mytag" in ret.extras, ret.extras
 
 
+def test_openmm_cmiles_gradient():
```

Comment:
This test is good, but I think it would be worthwhile constructing a test that succeeds past conformer generation. Can you add one that uses e.g. `qcng.get_molecule('eneyne')` with appropriate CMILES?